runtime/doc/options.txt

```diff
@@ -5783,7 +5776,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 			   *'statusline'* *'stl'* *E540* *E542*
 'statusline' 'stl'	string	(default empty)
-			global or local to window |global-local|
+			global or local to buffer |global-local|
```

Comment:
shouldn't be changed, "global or local to window" is correct.

Reply:
I can't explain the behavior in https://github.com/neovim/neovim/pull/11902  though. Is it possible to describe 'statusline' behavior in one sentence? Then we can add that here.

Reply:
But it  behaves just like any "global or local to window" option, and furthermore the behavior described in #11902 is even the behavior for `setlocal` for _any_ window option (global-window and window only differ with `setglobal`). This sounds like just another user who has experimented with window options for two minutes and then decided how window options "works" in a total sense and then expect documentation and other behaviors to be changed because their initial conclusion _must_ be correct and everything that contradicts it _must_ we wrong. we cannot make all these people happy. 

I agree that docs can be improved but this should be a _systematic_ effort (like a rewrite/restructuring of the local options section), and not by adding stray sentences here and there to try to appease one or another specific user who already "knows" how local options work.

Reply:
The "stray sentence" was because I've heard that 'statusline' is special somehow. Is it or isn't it?

Reply:
It used to be the _only_ window global-local option, but is no longer (should be 4-5 of them now). That's all that has been "special" about it.